### ISSUE TITLE
Nodejs 21 doesnt' work

### ISSUE THREAD

**Issue by munntjlx** [thomvaill/log4brains#117]
Mostly "tons of error messages" with node21. Wondering if person doing this has abandonded the project? Its GREAT software!

**Comment by ezavgorodniy** [comment]
Speaking of abandoning the project it's kind of duplicate of 

- https://github.com/thomvaill/log4brains/issues/113
- https://github.com/thomvaill/log4brains/discussions/108

node21 can be workarounded with adding a flag to your environment variable (NODE_OPTIONS=--openssl-legacy-provider) https://github.com/thomvaill/log4brains/pull/114

**Comment by nh8157** [comment]
Second this. Downgrading node to `20.11.1` fixed it for me.

**Comment by oobleck** [comment]
> Speaking of abandoning the project it's kind of duplicate of
> 
> * [Is this project DEAD? (canary issue) #113](https://github.com/thomvaill/log4brains/issues/113)
> * [Does this project need a new maintainer? #108](https://github.com/thomvaill/log4brains/discussions/108)
> 
> node21 can be workarounded with adding a flag to your environment variable (NODE_OPTIONS=--openssl-legacy-provider) #114

To be clear, this workaround is for a very specific issue. I don't believe this is the issue we are facing with node 21, nor does this workaround work for me on node 21.6.2.

`yarn log4brains preview` results in
```
~/data-table-adr/node_modules/esm/esm.js:1
const __global__ = <snip>;
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            

TypeError: Function.prototype.apply was called on undefined, which is a undefined and not a function
    at qo (~/workspace/data-table-adr/node_modules/esm/esm.js:1:223175)
    at Uo (~/workspace/data-table-adr/node_modules/esm/esm.js:1:222896)
    at $o (~/workspace/data-table-adr/node_modules/esm/esm.js:1:224488)
    at wu (~/workspace/data-table-adr/node_modules/esm/esm.js:1:227324)
    at Eu (~/workspace/data-table-adr/node_modules/esm/esm.js:1:227999)
    at Module.<anonymous> (~/workspace/data-table-adr/node_modules/esm/esm.js:1:295976)
    at n (~/workspace/data-table-adr/node_modules/esm/esm.js:1:279589)
    at Object.<anonymous> (~/workspace/data-table-adr/node_modules/log4brains/dist/log4brains:5:18)
    at Module._compile (node:internal/modules/cjs/loader:1378:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1437:10)

Node.js v21.6.2
[2K[1G[31merror[39m Command failed with exit code 1.
```

**Comment by thomvaill** [comment]
Fixed in [v1.1.0](https://github.com/thomvaill/log4brains/releases/tag/v1.1.0) 😃
